GitExtensions/Program.cs

```diff
@@ -48,6 +49,13 @@ private static void Main()
                     return UserEnvironmentInformation.GetInformation();
                 };
 
+                DiagnosticsClient.Initialize(ThisAssembly.Git.IsDirty);
+
+                //
+                Application.ApplicationExit += (s, e) => DiagnosticsClient.OnExit();
+                AppDomain.CurrentDomain.UnhandledException += (s, e) => DiagnosticsClient.Notify((Exception)e.ExceptionObject);
+                Application.ThreadException += (s, e) => DiagnosticsClient.Notify(e.Exception);
```

Comment:
Nitpick: What comment did you have in mind for line 54?

Reply:
Mmm... it was late in the evening... 😊 
Thanks for spotting